package.json

```diff
@@ -11,7 +11,7 @@
     "terminal-screenshot": "out/src/cli.js"
   },
   "scripts": {
-    "lint": "prettier . --check",
+    "lint": "eslint --max-warnings 0 ./ && prettier --check ./",
```

Comment:
Looks like this PR also added 3 outdated/legacy images to `tests/screenshots/`, perhaps from a rebase. I suggest removing them.